### ISSUE TITLE
Accessible Name for DatePicker

### ISSUE THREAD

**Issue by qazizaahirah** [OfficeDev/office-ui-fabric-react#9880]
When adding ariaLabel to DatePicker I am not able to see the accessible name getting reflected as accessible name 

Widget role: combobox
Design pattern:
Combo Box
Accessible name: (no value)

When I run the accessibility test (Custom Widget: Design Pattern) on https://developer.microsoft.com/en-us/fabric#/controls/web/datepicker

I find the same accessibility issue 

I am using office-ui-fabric-react@7.11.1

`          <div className="cat-left-data">
            <DatePicker
              id="Auto Publish Date"
              placeholder="Select Auto Publish date"
              ariaLabel="Select-publish-date"
              pickerAriaLabel="Select-a-date"
              onSelectDate={this._autoPublishDateOnChange}
              value={this.state.tagObject.autoPublishDate}
              disableAutoFocus={true}
            />
          </div>`

**Comment by joschect** [comment]
@qazizaahirah Thanks for submitting this ticket, is the accessible name something different than "aria-label" I see it showing up properly on the demo page that you linked.  
![image](https://user-images.githubusercontent.com/20801176/61664686-23f06e80-ac88-11e9-8be7-b96f9aeb0c36.png)

I also hear that the aria-label is read out correctly when using narrator in edge. Could you confirm that it's not actually read out?

**Comment by qazizaahirah** [comment]
Th reader is reading it all fine but the assessment for accessibility tool is failing. If you test Custom Widget: Design Pattern) on the tool you see that accessible name has no value
![image](https://user-images.githubusercontent.com/15657156/61665131-1e475880-ac89-11e9-9d9c-d4c96d556409.png)

**Comment by joschect** [comment]
@qazizaahirah What exactly does "accessibility name" correspond to? Which accessibility test are you running?

**Comment by qazizaahirah** [comment]
![image](https://user-images.githubusercontent.com/15657156/61667788-ccee9780-ac8f-11e9-9ec5-8b0dc396b154.png)
Here is the test that I am running

**Comment by joschect** [comment]
@qazizaahirah I'm still not sure which testing framework you're using. 

Looking at the aria specs it appears that for it to be a combobox that the element with role combobox should have the label on it. I think I know what to fix since.

**Comment by qazizaahirah** [comment]
I am using Accessibility insights for Web, the role to date picker is automatically assigned but the accessible name is missing besides aria label being there. This is either a office UI bug or a tool error. Can you confirm that this is not a bug from you guys. I can get in touch with the accessibility tool team if its not an error from fabric ui

**Comment by joschect** [comment]
@qazizaahirah I believe that this is a valid bug, according to the [aria specs](https://www.w3.org/TR/wai-aria-practices/#combobox) a combobox should have the label attached to the component that has `role='combobox'` on it. So I'm guessing this is popping up since the div container has 'role=combobox' but the input has the label.

**Comment by qazizaahirah** [comment]
Will you guys be addressing this bug?

**Comment by joschect** [comment]
@qazizaahirah Yes, we will go ahead and get it fixed.

**Comment by taparikh** [comment]
@joschect Hi! wanted to confirm when this PR with fix be merged into master ?

**Comment by taparikh** [comment]
@joschect Hi Jon, wanted to add another accessibility bug relating to the error message on datepicker and all other input types (dropdown, textfield) 

The error message does not have an accessible name. please find the screenshot below

![image](https://user-images.githubusercontent.com/51679982/62177555-2a22c280-b2fa-11e9-938f-dfec2c374b1b.png)

**Comment by joschect** [comment]
@taparikh It should be getting merged in a day or so.

That error message should go away once the fix is in. Per my previous comments the aria spec is clear about where the label should be going for comboboxes.

**Comment by taparikh** [comment]
@joschect  Hi Jon - I updated to the version you mentioned which has the fix - but I am still facing the issue. The DatePicker does not have an accessible name

**Comment by taparikh** [comment]
I see that the datepicker component does have an accessible name when checked in the inspect element. However the accessibility insights tool is still flagging them as an error. 

![image](https://user-images.githubusercontent.com/51679982/62984750-3a05d080-bde9-11e9-9112-3b480671dc43.png)

**Comment by taparikh** [comment]
@joschect  Wanted to confirm that this fix solves our accessible issue. Thanks!
